src/v0/destinations/google_adwords_enhanced_conversions/transform.js

```diff
@@ -35,7 +36,16 @@ const updateMappingJson = (mapping) => {
 const responseBuilder = async (metadata, message, { Config }, payload) => {
   const response = defaultRequestConfig();
   const { event } = message;
-  const filteredCustomerId = removeHyphens(Config.customerId);
+  const { subAccount } = Config;
+  let { customerId, loginCustomerId } = Config;
+  if (isNumber(customerId)) {
```

Comment:
does config support numeric data types ? I'm expecting regardless on user input, customerId always gets sent as string to transformer.

Reply:
The dynamic config allows us to expect numeric data from the payload. The bugsnag alert was caused by the dynamic config only.